src/engraving/libmscore/instrument.h

```diff
@@ -276,6 +276,30 @@ class PartChannelSettingsLink final : private ChannelListener
     friend void swap(PartChannelSettingsLink&, PartChannelSettingsLink&);
 };
 
+//---------------------------------------------------------
+//   Transposition
+//---------------------------------------------------------
+
+enum class TranspositionType
+{
+    Unknown,
+    Tuning,
+    Transposition,
+    Course
+};
```

Comment:
These are not types of transposition (e.g. "course" basically means "strings", like for a guitar), but there is no musical term to describe what they are actually types of. I used `<dropdownName meaning="...">` as deliberately vague terms, but I guess "dropdown" is a UI component so it's not really a suitable name for something in the instruments model.

How about we call the struct below `Trait` and the enum class `TraitType`? Trait is defined as:

> A distinguishing characteristic or quality

https://www.dictionary.com/browse/trait

You could make this change in `instruments.xml` too if you like. If so, just change the `update_instruments_xml.py` script to make it write `<traitName type="...">...</traitName>` instead of "dropdownName", etc.

Reply:
>How about we call the struct below Trait and the enum class TraitType?

Yes, I like this. Transposition was a temporary name for this structure :)